src/frontend/css/general.css

```diff
@@ -96,41 +164,29 @@ div.board-container {
   display: flex;
 }
 
-div.menu-container {
-  grid-area: right;
-  height: 100%;
-  width: 100%;
-}
+/* Control Panel */
 
-div.games-container {
-  height: 100%;
-  width: 100%;
-  align-items: center;
-}
-
-div.game-options {
-  width: 100%;
-  align-items: center;
-}
-
-div.challenges-container {
+div.menu-container {
+  grid-area: panel;
   height: 100%;
   width: 100%;
-  align-items: center;
 }
 
-div.settings-container {
-  overflow: auto;
+div.menu-container h4 {
+  position: relative;
+  left: 5px;
+  top: -10px;
+  font-weight: normal;
 }
 
 ul.game-list {
   width: 100%;
   margin-top: 1rem;
-  border-top: 1px solid #666666;
+  padding-bottom: 1rem;
```

Comment:
This fixes an issue where the drop-shadow below the last game in a list would be cut off.

Reply:
Big if true - well done